<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-P81C98J" ignored="0" name="Default test(failed)(failed)" tests="2" failures="2" timestamp="2022-05-20T11:44:31 IST" time="0.141" errors="0">
  <testcase name="test01" time="0.016" classname="Retryfailedtc.Test01">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Retryfailedtc.Test01.test01(Test01.java:12)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test01 -->
  <testcase name="test02" time="0.0" classname="Retryfailedtc.Test01">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Retryfailedtc.Test01.test02(Test01.java:19)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
</testsuite> <!-- Default test(failed)(failed) -->
